src/components/ChangeLocation/index.js

```diff
@@ -37,9 +38,14 @@ const ChangeLocation = ({
         keyboard={false}
       >
         <Modal.Header className={style.modalHeader}>
-          <Modal.Title className={style.modalTitle}>
-            Choose Location
-          </Modal.Title>
+          {modalHeaderTitle?.map((header, idx) => {
+            return (
+              <Modal.Title className={style.modalTitle} key={idx}>
+                {header.title}
+              </Modal.Title>
+            );
+          }
+          )}
```

Comment:
did you check the other pages using the ChangeLocation component as well? are they not affected with this change?